app/Nova/Filters/AdminInformationFilter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Filters;
+
+use Illuminate\Database\Eloquent\Builder;
+use Laravel\Nova\Filters\Filter;
+use Laravel\Nova\Http\Requests\NovaRequest;
+
+final class AdminInformationFilter extends Filter
+{
+    public $name = 'On hold';
```

Comment:
Can you use constants for these values? Especially the internal values "on_hold" and "not_on_hold".

Reply:
Here I fully agree that using a private constant does help 😊